docs/openapi.yaml

```diff
@@ -171,72 +226,264 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-        404:
-          description: Transfer ID not found
+        401:
+          description: Client is not authorized to access DTS
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-        5XX:
-          description: An unexpected error occurred
+              examples:
+                get-root:
+                  $ref: "#/components/examples/unauthorized-error"
+        404:
+          description: Transfer ID not found
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
-              examples:
-                internal-server-error:
-                  $ref: "#/components/examples/internal-server-error"
 
 components:
   schemas:
-    ServiceInfo:
+    Contributor:
       type: object
-      description: Service/API metadata
+      description: >
+        Represents a contributor to the resource. Contributors must have a
+        'contributor_type', either 'Person' or 'Organization', and a 'name'.
+
+        The 'credit_name' field is used to store the name of a person as it
+        would appear in a citation. If there is no 'credit_name' supplied, the
```

Comment:
Would be good if this could be some kind of auto-include so that it doesn't need to be edited if the KBCMSchema changes.

Reply:
Agree. That might actually mean autogenerating this file (which I'd be fine with, since this YAML syntax is not particularly cuddly anyway).

Reply:
See #55